lang/ui.en.json

```diff
@@ -71,6 +71,10 @@
     "defaultMessage": "Close",
     "description": "Close button text or label"
   },
+  "code-download-error": {
+    "defaultMessage": "Error downloading the target code",
+    "description": "Title for error message relating to project loading"
+  },
```

Comment:
One string added. Do not merge till strings clean up is merged

Reply:
I've now merged that